openmeter/ent/schema/price.go

```diff
@@ -0,0 +1,49 @@
+package schema
+
+import (
+	"entgo.io/ent"
+	"entgo.io/ent/schema/edge"
+	"entgo.io/ent/schema/field"
+	"entgo.io/ent/schema/index"
+
+	"github.com/openmeterio/openmeter/pkg/framework/entutils"
+)
+
+type Price struct {
+	ent.Schema
+}
+
+func (Price) Mixin() []ent.Mixin {
+	return []ent.Mixin{
+		entutils.IDMixin{},
+		entutils.NamespaceMixin{},
+		entutils.TimeMixin{},
+		entutils.CadencedMixin{},
+	}
+}
+
+func (Price) Fields() []ent.Field {
```

Comment:
pls review @tothandras @chrisgacsal

Reply:
Also is this specific to subscriptions or is this a general use table?